ui/client/components/graph/Graph.js

```diff
@@ -43,18 +43,19 @@ class Graph extends React.Component {
 		})
 
 		this.espGraphRef = React.createRef()
+
+		this.windowListeners = {
```

Comment:
Code formatting ;)

Reply:
Done. Are you sure we should leave new commit? :slightly_smiling_face: